.github/workflows/web.yaml

```diff
@@ -0,0 +1,44 @@
+name: package:web
+permissions: read-all
+
+on:
+  # Run CI on pushes to the main branch and on PRs.
+  push:
+    branches: [ main ]
+    paths:
+      - '.github/workflows/web.yaml'
+      - 'web/**'
+  pull_request:
+    paths:
+      - '.github/workflows/web.yaml'
+      - 'web/**'
+  schedule:
+    - cron: "0 0 * * 0"
+
+defaults:
+  run:
+    working-directory: web/
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sdk: [stable, beta, dev]
+        test_config: ['-p chrome', '-p chrome -c dart2wasm']
+
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+      - uses: dart-lang/setup-dart@e51d8e571e22473a2ddebf0ef8a2123f0ab2c02c
+        with:
+          sdk: ${{ matrix.sdk }}
+
+      - run: dart pub get
+      - run: dart format --output=none --set-exit-if-changed .
+        if: ${{ matrix.sdk == 'stable' }}
```

Comment:
We ran `dart format` with dev as well before. I'd remove this (and it's okay if it runs with beta).